templates/email/order/order_placed_notification.html.twig

```diff
@@ -0,0 +1,16 @@
+{% extends 'email/email-base.html.twig' %}
+
+{% block content %}
+    <p>
+        Hallo {{ order.requester.name }},
```

Comment:
@bartrail can you put in a decent script? Have the complete order at your disposal

Reply:
@DavidBadura @DanielBadura könnt Ihr mal drüber schauen, ich muss noch einige andere Mails basteln und würde gerne ne gute Grundlage haben. Ich hab mich erstmal mit nem speziellen Command für die Mails da rein gebastelt. Ich find die Controller aktuell etwas zu groß... das wird dann aber nen späteres Refactor, da fällt dann der der Mail Command weg, aber jetzt wie gesagt erstmal nur die Mailings rein.